### ISSUE TITLE
Can't add rust-mode to zeal-at-point-mode-alist using dotfile config

### ISSUE THREAD

**Issue by francoiswnel** [syl20bnr/spacemacs#6522]
I am not sure if this is user error, apologies if it is.

I am trying to add rust-mode to the zeal-at-point-mode-alist variable with `(add-to-list 'zeal-at-point-mode-alist '(rust-mode . "rust"))` in my user config, which is resulting in a variable is void error while starting spacemacs. I assume this is caused as the package is not loaded yet at that point.

I have opened an issue at the zeal-at-point repo, but it appears to be abandoned. Is this something that can be fixed in the spacemacs dash layer?

**Comment by NJBS** [comment]
Your assumption is correct, you can insure that the code is only evaluated after the package is loaded and the variable exists by modifying it to the following:

``` elisp
(with-eval-after-load 'zeal-at-point
  (add-to-list 'zeal-at-point-mode-alist '(rust-mode . "rust")))
```

**Comment by francoiswnel** [comment]
Thanks @NJBS, this worked. I have also shared your fix on the package repo.

I still think it's something that should be included by default if possible.

**Comment by francoiswnel** [comment]
This was fixed by [PR#20](https://github.com/jinzhu/zeal-at-point/issues/19) in zeal-at-point.
